### PR TITLE
Add android close button on native stack

### PR DESCRIPTION
**Pull request by osdnk**
image

image

image

### CODE REVIEW

**Comment by christianbaroni** [comment]
@osdnk wondering if we need this, I think it needs work visually if yes

people tend to use the device back button on android, right?

do gestures work currently on native stacks?

**Comment by osdnk** [comment]
@christianbaroni On new Androids there's no back button. Also, gestures are not working on a new version on Android (it's possible to make some weird interaction, but it doesn't look like a good pattern) and apps tend to have some way of navigating over the app using just an app interface. 

// cc @brunobar79

File: src/components/expanded-state/chart/ChartExpandedStateHeader.js [hunk 1]
```diff
@@ -88,7 +96,15 @@ export default function ChartExpandedStateHeader({
           shadow={coinIconShadow}
           symbol={asset?.symbol}
         />
-        <ChartContextButton asset={asset} color={color} />
+
+        {ios ? (
```
Comment (skylarbarrera) [inline src/components/expanded-state/chart/ChartExpandedStateHeader.js:100]: ios version is getting close button instead of contextButton

**Comment by brunobar79** [comment]
I'm not a big fan of adding the X on the right for all the screens TBH.

I think we should deal with it case by case.

For Settings and Asset Expanded State  I suggest we use the standard android navbar which is:
Back Arrow | Title left aligned (if Present)  | other icons right aligned (if present)


This is an example taken from the PlayStore App: 

Screen Shot 2020-10-29 at 1 47 58 AM

For the SendSheet, I'd prefer to keep it as a sheet.

But hey, this is just my personal opinion!

**Comment by osdnk** [comment]
Hmmm, okay, that might be a point

**Comment by brunobar79** [comment]
can we close this one?

**Comment by brunobar79** [comment]
Stack navigator pr took care of this

**Pull request closed without merging**